lib/manager/gradle-lite/index.ts

```diff
@@ -0,0 +1,9 @@
+import * as gradleVersioning from '../../versioning/gradle';
+
+export { extractAllPackageFiles } from './extract';
+export { updateDependency } from './update';
+
+export const defaultConfig = {
+  fileMatch: ['(^|/)gradle.properties$', '\\.gradle(\\.kts)?$'],
```

Comment:
I don't think this is accurate. You've written a groovy parser/lexer, but not a kotlin one.

Reply:
You're right. As I can see, lexer mostly is same, but we need to add couple more Kotlin syntax matches.

Reply:
What to do then? Mark this as draft until those are added, or remove the fileMatch?

Reply:
Converted to draft until make sure Kotlin is being handled fine too

Reply:
https://github.com/zharinov/intellij-scala-to-kotlin/pull/1/files